src/NuGet.Services.AzureSearch/Db2AzureSearch/NewPackageRegistrationProducer.cs

```diff
@@ -215,6 +217,24 @@ private async Task<IReadOnlyList<PackageRegistrationInfo>> GetPackageRegistratio
             }
         }
 
+        private bool ShouldSkipPackageRegistration(PackageRegistration packageRegistration)
+        {
+            if (_options.Value.SkipPackagePrefixes == null)
+            {
+                return false;
+            }
+
+            foreach (var skippedPackagePrefix in _options.Value.SkipPackagePrefixes)
```

Comment:
Do we care about performance here? Running every ID through a list of prefixes to ignore can be slow.

Reply:
The bottleneck for db2azuresearch is indexing data not producing it, so this change makes db2azuresearch signficantly faster on DEV. Also, this list will be `null` on INT and PROD and should have negligible impact.